core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -192,11 +192,14 @@ default void importTable(String tableName, String importDir)
    *
    * @param tableName
    *          Name of a table to create and import into.
+   * @param ic
+   *          ImportConfiguration for the table being created. If no configuration is needed pass
+   *          {@link ImportConfiguration#empty}
    * @param importDirs
    *          A set of directories containing the files copied by distcp from exportTable
    * @since 2.1.0
    */
-  void importTable(String tableName, Set<String> importDirs)
+  void importTable(String tableName, Set<String> importDirs, ImportConfiguration ic)
```

Comment:
Should the table name and import directories also be pushed into ImportConfiguration, so we're not splitting them up into an object with parameters, plus more parameters when the object is used?

Reply:
That's not a bad idea. My thinking here was that the table name and imporDirs are required parameters, where the ImportConfiguration holds all the optional parameters.

Reply:
The container object is last and should be able to be reused. This is consistent with other APIs in TableOperations.